tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -876,6 +883,7 @@ class DxcCompiler : public IDxcCompiler3,
           IFT(pResult->SetOutputObject(DXC_OUT_PDB, pDebugBlob));
         }
       }
+// SPIRV change ends
```

Comment:
it's better to put this on line 874 (after the #endif)

Reply:
Done, but I also changed line 876, which might be a minor.